### ISSUE TITLE
Context menu not working through keyboard

### ISSUE THREAD

**Issue by JasonYeMSFT** [microsoft/AzureStorageExplorer#1825]
It seems that under circumstances where the containers have a lot of entries, opening context menu via the menu key or Shift-F10 doesn't work.

This needs to be fixed asap since it affects accessibility.
Extra finding: Ctrl-5 can be a current work around.

**Comment by JasonYeMSFT** [comment]
- [ ] Blob Container context menu
- [ ] ADLS Gen 2
- [ ] Fileshare
- [ ] Queue
- [ ] Table
- [ ] ADLS Gen 1
- [x] Managed Disks

**Comment by MRayermannMSFT** [comment]
Jason merged a fix into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.
